### DISCUSSION TITLE
Try harder OCR detection for some labels in case of OCR typos

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1212] [Ideas]
```ACRICVLTURE\nSIOLOGIQUE\n1\nCERTIFIÉ PAR FR-BID-T0``` 

https://static.openfoodfacts.org/images/products/376/012/121/3259/1.json

**Comment by raphael0202** [comment]
I'm not sure the response lies on the regex side, more on the OCR side (we may run the OCR on the same image once Google Cloud release a new version of its model)
